README.md

```diff
@@ -1,39 +1,43 @@
-# Todo App
+# Weather App
 
-#### This is a ToDo list app, save all your pending tasks to remember, save all your memos and don't forget any important activity. Use this app as you ever you want. 
-
-<div align="center"><img src="https://user-images.githubusercontent.com/24902525/89691065-a668c200-d8cd-11ea-932c-c64cb9139495.png" alt="image" width="900" /></div>
+#### Check the weather in your current location, open the app. Enter a city name and see its weather description.Beautiful design and displaying a photo according to the entered city. It changes background color according to weather (clouds, sunny, raining, and shower rain). Toggles between Celsius and Fahrenheit 
 
+<div align="center"><img src="https://user-images.githubusercontent.com/24902525/90455284-5357f180-e0bb-11ea-9c14-6132f944294c.gif" alt="image" width="900" /></div>
 
+**Note:** Not recommendable for smartphones yet
 
 ## Built With 
 
 - JavaScript ES6 Modules
-- HTML5
+- HTML5s
 - CSS3
 - Webpack
 - Babel
+- Open weather map [API](http://api.openweathermap.org)
+- Ip Stack get current location by IP [API](http://api.ipstack.com)
+- Teleport for city images [API](https://api.teleport.org/api/urban_areas/slug:san-francisco/images)
+- Country flag  [service](https://www.countryflags.io/EC/flat/64.png)
 
-## Live Demo
 
-[:globe_with_meridians::computer:](https://rawcdn.githack.com/NewIncome/mv_todo-list/360b90c602894736e0f744429a057caff0ed5042/index.html)
+## Live Demo
 
+[:globe_with_meridians::computer:](http://weather.bertiltandayamo.me/)
 
-### Usage
 
-- Add a project (title and description)
-- Click on the projects and add n number of tasks you want
-**Note:** All your task will be saved on the browser local storage
+## Usage
+- Clone the [project 📝](https://github.com/bertil291utn/weather-app.git)
+- Install all dependencies `npm i`
+- Get all your APIs and add it to `.env` file
```

Comment:
- [ ] This instruction is too vague. 
Specify the name that each API key should have a from which site to get each needed key.
The issue from the last review was related to that, make sure the information on how to run it locally is as explicit as possible.

Remember: Documentation is key!

Reply:
Yes, I didn't know what exactly as waskin gon previous review , and it was my error.
Thanks for the feedback